databases/weaviate/docker/chatbot/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM python:3.12-slim-bookworm
```

Comment:
Could you create a new workflow for this weaviate samples, that has 4 steps:
- 2x `docker build` steps for the two containers
- 2x `terraform validate` steps for the two terraform directories

You can find an example here: https://github.com/GoogleCloudPlatform/kubernetes-engine-samples/blob/main/.github/workflows/cost-optimization-gke-vpa-recommendations-ci.yml

Reply:
yep, sure, will add

Reply:
done